src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AvoidMultipleEnumerations/AvoidMultipleEnumerations.DataFlowOperationVisitor.cs

```diff
@@ -37,52 +40,245 @@ public override GlobalFlowStateDictionaryAnalysisValue VisitLocalReference(ILoca
                 return VisitLocalOrParameter(operation.Local.Type?.OriginalDefinition, operation, value);
             }
 
-            private GlobalFlowStateDictionaryAnalysisValue VisitLocalOrParameter(ITypeSymbol? typeSymbol, IOperation operation, GlobalFlowStateDictionaryAnalysisValue defaultValue)
+            private GlobalFlowStateDictionaryAnalysisValue VisitLocalOrParameter(ITypeSymbol? typeSymbol, IOperation parameterOrLocalReferenceOperation, GlobalFlowStateDictionaryAnalysisValue defaultValue)
```

Comment:
> parameterOrLocalReferenceOperation

Add an assert for the possible operation kinds at start of the method?

Reply:
Fixed